include/notcurses/notcurses.h

```diff
@@ -2,7 +2,17 @@
 #define NOTCURSES_NOTCURSES
 
 #include <time.h>
+
+#if !defined(__APPLE__)
+// Apple clang supports C11, but it does not have uchar.h
```

Comment:
supposedly uchar.h is present on mac os x following 10.3? no?

Reply:
No sir, no uchar.h... :)

Reply:
:(